src/pages/Home.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './Home.css';
+
+const Home = () => {
+  return (
+    <div className='home-page'>
+      <h2>Welcome to Pokemon Tinder!</h2>
+      <p></p>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
+      <br/>
```

Comment:
There are a lot of break tags here, I would recommend updating this with some css.